wiki/Twitter/id.md

```diff
@@ -1,18 +1,24 @@
----
-outdated: true
-outdated_since: a6c6c6b27315fdf9e68f709dab2a08b085428e3b
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
+| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Membantu kamu terkait masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```

Comment:
```suggestion
| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Diperuntukkan untuk mengatasi masalah mengenai akun maupun komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
```

Reply:
gonna be sticking consistently with previous wording as well to helping people understand